pycodestyle.py

```diff
@@ -2114,7 +2114,11 @@ def check_ast(self):
         except (ValueError, SyntaxError, TypeError):
             return self.report_invalid_syntax()
         for name, cls, __ in self._ast_checks:
-            checker = cls(tree, self.filename)
+            checker = None
+            if len(_get_parameters(cls.__init__)) == 3:
+                checker = cls(tree, self.filename)
+            elif len(_get_parameters(cls.__init__)) == 4:
+                checker = cls(tree, self.filename, self.lines)
```

Comment:
pycodestyle doesn't use this form, this also potentially leaves `checker` as none